src/app.ts

```diff
@@ -29,7 +29,7 @@ const cli = new Cli({
         reg.setId(AppServiceRegistration.generateToken());
         reg.setHomeserverToken(AppServiceRegistration.generateToken());
         reg.setAppServiceToken(AppServiceRegistration.generateToken());
-        reg.setSenderLocalpart(config.bot_username);
+        reg.setSenderLocalpart("slackbot");
```

Comment:
Surely this wants to be configurable?

Reply:
These are just the defaults, you are expected to go configure them yourself.